sanitiser/_truthy.js

```diff
@@ -0,0 +1,9 @@
+function isTruthy(val) {
```

Comment:
this could handle the checking and resolution of default value as well. accept an optional `default_value` param along with `val` and do the checking in here for `val` === `undefined`.